### ISSUE TITLE
Map not downloading data

### ISSUE THREAD

**Issue by LordDonkey** [DSheirer/sdrtrunk#570]
The map is not downloading the data from openstreetmap. I can open the link that it fails to download in my browser and see the image. Could something be blocking the connection on my system? 

21:33:51.232 ERROR o.j.s.m.AbstractTileFactory - Failed to load a tile at url: http://tile.openstreetmap.org/3/6/6.png, retrying
java.io.IOException: Server returned HTTP response code: 403 for URL: http://tile.openstreetmap.org/3/6/6.png
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
        at java.base/java.net.URL.openStream(Unknown Source)
        at org.jdesktop.swingx.mapviewer.AbstractTileFactory$TileRunner.cacheInputStream(AbstractTileFactory.java:391)
        at org.jdesktop.swingx.mapviewer.AbstractTileFactory$TileRunner.run(AbstractTileFactory.java:341)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)

**Comment by kazoni** [comment]
When you open in a browser, it's redirecting to http**s**://tile.openstreetmap.org/3/6/6.png

**Comment by LordDonkey** [comment]
Yeah, I noticed that. Could be that the tile server is trying to force SSL/TLS? I have very limited Java experience, I took a semester of computer science a few years ago in high school, but I managed to get the map working. I went into the OSMTileFactoryInfo.java and replaced http://tile.openstreetmap.org with http://a.tile.openstreetmap.fr/hot, recompiled and the map works. Not sure if that is allowed to be used but it is a workaround.
